modules/network/nsgs.tf

```diff
@@ -364,6 +364,30 @@ resource "oci_core_network_security_group_security_rule" "int_lb_egress" {
   count = var.load_balancers == "internal" || var.load_balancers == "both" ? length(local.int_lb_egress) : 0
 }
 
+resource "oci_core_network_security_group_security_rule" "int_lb_ingress" {
+  network_security_group_id = oci_core_network_security_group.int_lb[0].id
+  description               = "Allow stateful ingress from ${element(element(local.internal_lb_allowed_cidrs_and_ports, count.index), 0)} on port ${element(element(local.internal_lb_allowed_cidrs_and_ports, count.index), 1)}"

```

Comment:
The description generated is not correct . When we provide internal_lb_allowed_cidrs and internal_lb_allowed_ports the description is not correct. please review.